src/foam/nanos/extraconfig/Config.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.extraconfig',
+  name: 'Config',
+  properties: [
```

Comment:
add documentation:

Could you have a more informative name than just 'Config'?

Reply:
fixed